extra-wallpaper/aosc-community-wallpapers-2020/spec

```diff
@@ -0,0 +1,4 @@
+VER=1.0.0
+GITTAG=20210105
```

Comment:
Merge these versions now that you separated the make utility?

Reply:
Now using date as VER